python/rikai/torch/data.py

```diff
@@ -131,3 +135,29 @@ def __iter__(self) -> Generator[Dict, None, None]:
             yield self.collate_fn(batch)
         prefetch_thd.join()
         q.join()
+
+
+def make_video_sampler(sample_gen, video_column_name=None):
```

Comment:
Add doc here? how should user expect to use it

Reply:
I will add it there. Current usage is in the notebook i added. I can move some code snippets over.